src/app/collective-rewards/leaderboard/LeaderBoard.tsx

```diff
@@ -55,12 +56,24 @@ export const LazyRewardCell = memo(RewardCell, ({ rewards: prevReward }, { rewar
   prevReward.every((reward, key) => reward.fiat.value === nextReward[key].fiat.value),
 )
 
-export const BuilderNameCell = ({ builderName }: { builderName: string }) => {
+export const BuilderNameCell = ({ builderName, address }: { builderName: string; address: string }) => {
   return (
     <TableCell className={cn(tableHeaders[0], 'border-solid')}>
       <div className="flex flex-row gap-x-1">
         <Jdenticon className="rounded-md bg-white" value={builderName} size="24" />
-        <AddressOrAlias addressOrAlias={builderName} className="text-sm" />
+        <Popover
+          content={
+            <div className="text-[12px] font-bold mb-1">
+              <p data-testid="builderAddressTooltip">{reduceAddress(address)}</p>
```

Comment:
I don't like showing the reduced address in the tooltip while we are showing the reduced address in the component as well.

Reply:
don't ;)

Reply:
maybe only show the popover when necessary. See [my popover change here](https://github.com/RootstockCollective/dao-frontend/pull/339/files#diff-59b5b3fac0365b31cf7988a3e70daa514a2005729867f9b9425f4e96efce8721R57)

Reply:
I agree, let me introduce that :), like just when is a builder name :)